src/app/node-data/component.ts

```diff
@@ -50,6 +50,7 @@ enum Controls {
   ProviderBasic = 'providerBasic',
   ProviderExtended = 'providerExtended',
   Kubelet = 'kubelet',
+  OperatingSystemProfile = 'operating system profile',
```

Comment:
Avoid spacing in control name see above controls. Use `camelCasing` syntax
e.g`RhelSubscriptionManagerPassword,OperatingSystem`